schema.json

```diff
@@ -82,6 +82,9 @@
                     "type": "string",
                     "enum": ["client", "server", "all"]
                 },
+                "id": {
+                    "type": "string"
+                },
```

Comment:
should we add a regex rule that ensures that an integer is provided?

Reply:
This is part of validation I assume. So IMHO, it is better to address this in the validation PR (WIP)

Reply:
It is actually a schema property and we do similar things in many existing schemas.  For example:

https://github.com/perftool-incubator/rickshaw/blob/master/schema/benchmark.json#L12

Reply:
> It is actually a schema property and we do similar things in many existing schemas. For example:
> 
> https://github.com/perftool-incubator/rickshaw/blob/master/schema/benchmark.json#L12

something like...
"pattern": "^[0-9]+$"